DeBERTa/apps/sequence_classification.py

```diff
@@ -68,4 +68,4 @@ def forward(self, input_ids, type_ids=None, input_mask=None, labels=None, positi
         label_confidence = 1
         loss = -((log_softmax(logits)*labels).sum(-1)*label_confidence).mean()
 
-    return (logits,loss)
+    return (loss, logits)
```

Comment:
During onnx conversion had error message requiring loss to be a first scalar in tuple